### PR TITLE
merge june release back

### PR DESCRIPTION
**Pull request by DavidKutu**
-   [ ] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [ ] Title summarizes what is changing.
-   [ ] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for enhancements.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,103 @@
...
+   ([#4585](https://github.com/Microsoft/vscode-jupyter/issues/4585))
+1. Limit languages dispalyed in the Cell language picker to languages supported by the kernel.
+   ([#5580](https://github.com/Microsoft/vscode-jupyter/issues/5580))
+1. Move native notebooks cell toolbar to the left by default.
+   ([#5605](https://github.com/Microsoft/vscode-jupyter/issues/5605))
+1. Display modal dialog box (so users don't miss this) when IPyKernel (or Jupyter) is missing (required to run Python in Interactive Window or Notebooks).
+   ([#5798](https://github.com/Microsoft/vscode-jupyter/issues/5798))
+1. Add support for [Virtual Workspaces](https://github.com/microsoft/vscode/wiki/Virtual-Workspaces).
+   ([#5803](https://github.com/Microsoft/vscode-jupyter/issues/5803))
+1. Losslessly compressed PNG images to save ~20KB.
```
Comment (rchiodo) [inline CHANGELOG.md:19]: I believe this needs a thank you?

**Review by rchiodo — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (DavidKutu) [reply CHANGELOG.md:19]: it is there, line 84

Comment (DavidKutu) [reply CHANGELOG.md:19]: oh I see what you mean

Comment (rchiodo) [reply CHANGELOG.md:19]: No an individual contributor thank you for this submission.

Comment (DavidKutu) [reply CHANGELOG.md:19]: I'll put it on the release branch, and it will be merged after we release

File: .github/workflows/build-test.yml [hunk 2]
```diff
@@ -70,11 +70,11 @@ jobs:
           echo "vsix_name=ms-toolsai-jupyter-insiders.vsix" >> $GITHUB_ENV
           echo "test_matrix_os=[\"ubuntu-latest\", \"windows-latest\"]" >> $GITHUB_ENV
 
-      - name: insiders channel
-        # Scheduled builds will publish Insider builds.
-        if: github.event_name == 'schedule' && github.ref == 'refs/heads/main'
-        run: |
-          echo "release_channel=insider" >> $GITHUB_ENV
+    #   - name: insiders channel
```
Comment (IanMatthewHuff) [inline .github/workflows/build-test.yml:73]: 👍 just make sure that we turn it back on with the versions updated when new insiders engine version is out.

**Review by IanMatthewHuff — APPROVED** [review]

File: .github/workflows/build-test.yml (hunk 2 above)
Comment (DavidKutu) [reply .github/workflows/build-test.yml:73]: Will not forget
https://github.com/microsoft/vscode-jupyter/issues/6195

**Pull request merged**